code/styles/gridFlexbox.css

```diff
@@ -0,0 +1,85 @@
+/* DEFAULT size - Mobile/Small Devices*/
+
+/* ======= LAYOUT (GRID&FLEX) ======= */
+body {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px 20% auto auto auto 200px;
+  grid-template-areas:
+    "header"
+    "hero"
+    "news-boxes"
+    "news-column"
+    "news-list"
+    "footer";
+  gap: 5px 5px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 10px;
+  padding-right: 10px;
+  width: 740px;
```

Comment:
This width is causing a horizontal scroll on mobile size. In responsive design we want to avoid that:

<img width="531" alt="Skärmavbild 2021-01-29 kl  14 43 39" src="https://user-images.githubusercontent.com/51529201/106282078-6234f980-6240-11eb-9179-e8c81129dc59.png">